### PR TITLE
Added support for object bag, closes #212

### LINKED ISSUE #212: New bag
A new bag has been added for furniture. New tab needs to be added to the GUI and adding the bag to save/load.

### PR DESCRIPTION
**Pull request by j4ck-no1r**

### CODE REVIEW

File: starcheat/gui/mainwindow.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def __init__(self):
 
         # set up bag tables
         bags = ("wieldable", "head", "chest", "legs", "back", "main_bag",
-                "action_bar", "tile_bag", "essentials", "mouse")
+                "action_bar", "object_bag", "tile_bag", "essentials", "mouse", "object_bag")
```
Comment (chrmoritz) [inline starcheat/gui/mainwindow.py]: adding `"object_bag"` a second time here is redundant

**Comment by chrmoritz** [comment]
LGTM, except one style nit :thumbsup:

**Comment by j4ck-no1r** [comment]
Whoops! Fixed that.

**Comment by LoPhatKao** [comment]
nice :D 
hopefully implemented soon

**Comment by wizzomafizzo** [comment]
You guys know anything about the bug with generated items resetting themselves after editing? Has it fixed itself now people's players have converted/settled or is it still out of sync and messing it up?

**Comment by j4ck-no1r** [comment]
It looks like non-generated weapons reset themselves. Dunno why, but I don't think it's an issue with Starcheat.

**Comment by wizzomafizzo** [comment]
@chrmoritz you got any objection to doing a new release?

i don't have anything else to add, i was just gonna test it works on windows and do one just so we can get this patch out

idk what's going on with the weapons resetting stuff. i'll do one last check on the item version ids but it's sounding like chucklefish have just locked down editing on non-generated items

**Comment by wizzomafizzo** [comment]
@j4ck-no1r looks like I've doubled up on a bunch of work you did with my latest commit sorry. I'll try get them merged properly.

**Pull request merged**